extensions/amp-story/1.0/amp-story-draggable-drawer.js

```diff
@@ -379,6 +385,17 @@ export class DraggableDrawer extends AMP.BaseElement {
           ? this.open()
           : this.close_();
       }
+
+      return;
+    }
+
+    if (
+      this.openThreshold_ !== null &&
+      this.state_ === DrawerState.DRAGGING_TO_OPEN &&
+      swipingUp &&
+      -deltaY > this.openThreshold_
```

Comment:
Are these conditions specifically for the remote URL case? if so, should we move them to a separate method so it's easier to know?